doc/proposal/in-progress/07-signing.md

```diff
@@ -0,0 +1,230 @@
+
+## Digests
+
+Digests are used for componentReferences and resources to indicate a hash for their content.
+
+Fields:
+digest:
+  - hashAlgorithm: Hash algorithm used
+  - normalisationAlgorithm: normalisation Algorithm, defines which subset of the data is used as normalised representation
+  - value: the digest value for the normalised data as defined by normalisationAlgorithm with the hash algorithm defined by normalisationAlgorithm
+
+## Signatures
+
+The component descriptor may have a list of signatures on root level. A signature can be identified by its name. It follows this schema:
+```
+signatures:
+  - name: name of the signature. Used to identify the signature when verifying.
+    digest: as defined, contains the digest of the normalised component-descriptor (the signing-relevant subset of the component-descriptor)
+    signature:
+        algorithm: signature algorithm used
+        value: the signature in the format as defined in mediaType
+        mediaType: defines the format of the signature.value field
+```
+
+## Normalisation CD
+
+The normalisation of the component-descriptor describes the process of generating a normalised component-descriptor. A normalised component-descriptor is a subset of the component-descriptor of signing-relevant properties. 
+
+### Requirements
+All component-references have a digest. Resources are required to have digest, except their access is null or the access.type is 'None'. If a digest-containing entry (= component reference or resource) already contain a digest during signing, the normalisation process must be aborted if the digest mismatches the calcuated digest for the entry.
+
+### Exclude Resource from Normalisation/Signing
+If a resource should not be part of the normalisation and later signing, the resource needs a special digest in the following format:
+digest:
+  hashAlgorithm: NO-DIGEST
+  normalisationAlgorithm: EXCLUDE-FROM-SIGNATURE
+  value: NO-DIGEST
+
+### Properties for normalisation
+
+The following properties are part of the normalised component-descriptor:
+```
+meta
+    schemaVersion
+
+component
+    name
+    version
+    provider
+
+    componentReferences
+        name
+        componentName
+        version
+        extraIdentity
+        digest:
+            hashAlgorithm
+            normalisationAlgorithm
+            value
+    resources
+        name
+        version
+        extraIdentity
+        type 
+        relation
+        digest:
+            hashAlgorithm
+            normalisationAlgorithm
+            value
+```
+Exceptions:
+
+1. If resource.Access is emtpy or resource.access.type is None, no digest field is required. Then, only name, version, extraIdentity, type and relation is used for the normalisation.
+
+### Properties **NOT** part of normalisation:
+
+component
+    repositoryContexts
+    labels
+    sources
+    componentReference:
+        labels
+    resource:
+        labels
+        access
+signatures
```

Comment:
Could you put this in a code block? Currently, the formatting is lost, so that all is displayed in one line.